bittensor/_neuron/text/core_validator/__init__.py

```diff
@@ -553,22 +577,22 @@ def run_epoch( self ):
             weights=sample_weights.detach().to('cpu'),
             netuid = self.config.netuid,
             wallet=self.wallet,
-            version_key=bittensor.__version_as_int__, # TODO: correct?
+            version_key=1,
```

Comment:
what is this about?

Reply:
The version key should be the bittensor version. Basically, the chain will check if the version key matches the latest bittensor version. However, that is not working right now, so it is set to 1

Reply:
Then are we keeping it as 1?

Reply:
For now, yes. I do not believe that feature is turned on yet.

Reply:
Okay! Just reverted it to 1:)